README.md

```diff
@@ -33,13 +33,13 @@ READMEs that explain specific DSCP projects:
 These repositories contain code being actively maintained as part of the DSCP project.
 
 | Repository                                                                         | Description                                                                   |
-| ---------------------------------------------------------------------------------- | ----------------------------------------------------------------------------- |
+| ---------------------------------------------------------------------------------- | ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------ | ------------------------------------------------ |
 | [dscp-documentation](https://github.com/digicatapult/dscp-documentation)           | Documentation for DSCP                                                        |
 | [dscp-flux-infra](https://github.com/digicatapult/dscp-flux-infra)                 | Flux repo to bring up DSCP with Kubernetes                                    |
 | [dscp-node](https://github.com/digicatapult/dscp-node)                             | The blockchain node for DSCP, built with Substrate                            |
 | [dscp-ipfs](https://github.com/digicatapult/dscp-ipfs)                             | IPFS node for DSCP                                                            |
 | [dscp-api](https://github.com/digicatapult/dscp-api)                               | API to create/retrieve tokens on `dscp-node` and files on `dscp-ipfs`         |
-| [dscp-identity-service](https://github.com/digicatapult/dscp-identity-service)     | API for managing chain member identities in DSCP                              |
+| [openapi](https://github.com/digicatapult/openapi-merger)                          | Amalgamated API specs for OpenAPI                                             | [dscp-identity-service](https://github.com/digicatapult/dscp-identity-service) | API for managing chain member identities in DSCP |
```

Comment:
openapi -> openapi-merger. Looks like a new line is needed here too cause `dscp-identity-service` is on the same line